src/cursor/cur_backup_incr.c

```diff
@@ -144,14 +157,21 @@ __curbackup_incr_next(WT_CURSOR *cursor)
              */
             WT_ERR(__curbackup_incr_blkmod(session, btree, cb));
             /*
-             * If there is no block modification information for this file, this is a newly created
-             * file without any checkpoint information. Return the whole file information.
+             * If there is no block modification information for this file, it's either a newly
+             * created file without any checkpoint information, a file created and checkpointed
+             * before backups were configured and not subsequently modified, or a file that has not
+             * been modified since the last incremental backup. In the first case, we return the
+             * whole file information. We ignore the file in the second and third, as these files
+             * were copied in the initial full backup or a previous incremental backup.
```

Comment:
Can this comment be reformatted (you'll have to see how to prevent clang-format from redoing it though) so that it reads something like:
There are three cases where we may not have block modification information and we do different things:
1. Newly created file without checkpoint information. Return the whole file information.
2. File created and checkpointed before incremental backups were configured. Return no file information as it was copied in the initial full backup.
3. File that has not been modified since the previous incremental backup. Return no file information as there is no new information.

Reply:
Done.